### PR TITLE
Batch labels and apply labels to all new tags

### PR DESCRIPTION
**Pull request by josephschorr**
Fixes https://issues.redhat.com/browse/PROJQUAY-279

### CODE REVIEW

**Comment by josephschorr** [comment]
@alecmerdler @kurtismullins Please take another look

File: data/model/oci/label.py [hunk 1]
```diff
@@ -84,6 +102,44 @@ def create_manifest_label(manifest_id, key, value, source_type_name, media_type_
...
+
+    # Ensure the manifest exists.
+    try:
+        manifest = Manifest.select(Manifest).where(Manifest.id == manifest_id).get()
+    except Manifest.DoesNotExist:
+        return None
+
+    label_ids = list()
+    with db_transaction():
+        # Create the labels. We do this non-batch-wise because we need the returned IDs.
```
Comment (kleesc) [inline data/model/oci/label.py:123]: Heh, postgres actually has a `RETURNING` clause to return data on bulk inserts.

Comment (josephschorr) [reply data/model/oci/label.py:123]: I wish we could use it :/

**Comment by kurtismullins** [comment]
@josephschorr Can you rebase this? Thanks!

**Comment by josephschorr** [comment]
PR has drifted too far from master. Will manually merge and reopen in a new branch.

**Pull request closed without merging**